composer.json

```diff
@@ -16,25 +16,24 @@
         "jms/metadata": "~1.1",
         "jms/parser-lib": "1.*",
         "phpcollection/phpcollection": "~0.1",
-        "doctrine/annotations": "1.*",
-        "doctrine/instantiator": "~1.0.3"
+        "doctrine/annotations": "^1.0",
+        "doctrine/instantiator": "^1.0.3"
     },
     "conflict": {
         "twig/twig": "<1.12"
     },
     "suggest": {
         "symfony/yaml": "Required if you'd like to serialize data to YAML format."
     },
-    "minimum-stability": "dev",
     "require-dev": {
         "twig/twig": "~1.12|~2.0",
         "doctrine/orm": "~2.1",
-        "doctrine/phpcr-odm": "~1.0.1",
-        "jackalope/jackalope-doctrine-dbal": "1.0.*",
+        "doctrine/phpcr-odm": "^1.3",
+        "jackalope/jackalope-doctrine-dbal": "^1.0",
         "propel/propel1": "~1.7",
         "symfony/yaml": "2.*",
         "symfony/translation": "~2.0",
-        "symfony/validator": "~2.0",
+        "symfony/validator": "~2.0.0|~2.1.0",
```

Comment:
i agree that here at least 2.3 should be supported, but this requires fixing all the tests that are using `ConstraintViolation` in https://github.com/schmittjoh/serializer/blob/master/tests/JMS/Serializer/Tests/Serializer/BaseSerializationTest.php

since `ConstraintViolation` is not backward compatible from 2.1 to 2.3